src/Pact/Types/SigData.hs

```diff
@@ -66,28 +74,33 @@ data SigData a = SigData
   , _sigDataCmd :: Maybe a
   } deriving (Eq,Show,Generic)
 
+sigDataProperties :: ToJSON a => JsonMProperties (SigData a)
+sigDataProperties o = mconcat
+  [ "hash" .= _sigDataHash o
+  , "sigs" .= HM.fromList (bimap unPublicKeyHex (fmap _usSig) <$> _sigDataSigs o)
+    -- FIXME fix order independently of hashable package
```

Comment:
Use `LegacyHashMap` for backward compatible behavior. Use list for pairs for behavior that complies with the comment on the `_sigDataSigs` constructor field.